lib/common/theme/text_styles.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter/material.dart';
+
+TextStyle onPrimarySubtitle(BuildContext context) {
+  return TextStyle(
+    fontSize: Theme.of(context).textTheme.headlineLarge?.fontSize,
```

Comment:
I suppose, it happens because of null safety, which  supports only from sdk version 2.12._
limitations are set in pubspec.yaml as:
environment:
  sdk: ">=2.16.2 <3.0.0"